assets/blocks/course-outline/lesson-block/settings.js

```diff
@@ -3,25 +3,47 @@ import {
 	InspectorControls,
 	PanelColorSettings,
 } from '@wordpress/block-editor';
+import { PanelBody, FontSizePicker } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
 
 /**
  * Inspector controls for lesson block.
  *
- * @param {Object}   props
- * @param {Object}   props.backgroundColor
- * @param {Object}   props.textColor
- * @param {Function} props.setTextColor
- * @param {Function} props.setBackgroundColor
+ * @param {Object}   props                    Component props.
+ * @param {Object}   props.backgroundColor    The lesson title background color.
+ * @param {Object}   props.textColor          The lesson title color.
+ * @param {Function} props.setTextColor       Callback method to set the lesson title color.
+ * @param {Function} props.setBackgroundColor Callback method to set the background color.
+ * @param {Function} props.fontSize           The font size of the lesson title.
+ * @param {Function} props.setAttributes      Callback method to set the lesson title font size.
  */
 export function LessonBlockSettings( {
 	backgroundColor,
 	textColor,
 	setTextColor,
 	setBackgroundColor,
+	fontSize,
+	setAttributes,
 } ) {
+	const { fontSizes } = useSelect( ( select ) =>
+		select( 'core/block-editor' ).getSettings()
+	);
+
 	return (
 		<InspectorControls>
+			<PanelBody
+				title={ __( 'Typography', 'sensei-lms' ) }
+				initialOpen={ true }
```

Comment:
Do we want this one open by default? Looks like it's open for Heading blocks, but maybe it's not as important for Lessons.

Reply:
Fixed in 6e94ee3a9a3f558776dae3b9b272b4e616450696